<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHome</class>
 <widget class="QWidget" name="frmHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>451</width>
     <height>156</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/btn_box.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDev">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>451</width>
     <height>156</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPara">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>451</width>
     <height>156</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/btn_para.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHistory">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>451</width>
     <height>156</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
